src/Components/Form/FormFields/FormField.tsx

```diff
@@ -41,7 +41,7 @@ const FormField = (props: {
     props.props;
 
   return (
-    <div className={className}>
+    <div className={className} id={id}>
```

Comment:
The ID is supposed to be used by the actual field component. Else we'll end up having the same id for the div and the component wrapped by the FormField

```suggestion
    <div className={className}>
```